lookup/localpgp.go

```diff
@@ -29,7 +29,15 @@ type LocalPGPService struct {
 // NewLocalPGPService creates a new LocalPGPService if it finds a local
 // public keyring; otherwise it bails.
 func NewLocalPGPService() (*LocalPGPService, error) {
-	ringfile := path.Join(os.Getenv("HOME"), ".gnupg", "pubring.gpg")
+
+	home := os.Getenv("HOME")
+
+	// Check if an override for GNUPG home is set
+	if os.Getenv("GNUPGHOME") != "" {
+		home = os.Getenv("GNUPGHOME")
+	}
+
+	ringfile := path.Join(home, ".gnupg", "pubring.gpg")
```

Comment:
If `$GNUPGHOME` is set, `ringfile` here would be `$GNUPGHOME/.gnupg/pubring.gpg`. Is that the standard usage? Reading the `gpg` man page, it looks like `$GNUPGHOME` is equivalent to `$HOME/.gnupg`, i.e. the top-level GnuPG directory:

> --homedir dir
>        Set the name of the home directory to dir. If this option is not used, the home directory defaults to `~/.gnupg`.  It is only recognized when given on the command line.  It also overrides any  home  directory  stated  through  the  environment  variable `GNUPGHOME`